tests/sentry/api/endpoints/test_organization_codeowners_associations.py

```diff
@@ -105,19 +105,14 @@ def test_response_data_is_correct(self):
             in response.data[self.project_2.slug]["errors"]["teams_without_access"]
         )
 
-    def test_no_access(self):
+    def test_member_can_access(self):
         """
-        Tests that users without the 'org:integrations' scope (i.e. Members) cannot access this endpoint.
+        Tests that users without the 'org:read' scope (i.e. Members) can access this endpoint.
```

Comment:
I would like to know the difference between `org:read` vs `org:write` to feel comfortable about giving `org:read` access to the endpoints.